air/src/stack/system_ops/mod.rs

```diff
@@ -104,3 +104,17 @@ pub fn enforce_fmpupdate_constraints<E: FieldElement>(
 
     1
 }
+
+/// Enforces constraints of the CLK operation. The CLK operation pushes the current cycle number to
+/// the stack. Therefore, the following constraints are enforced:
+/// - The first element in the next frame should be equal to the current cycle number. s0` - (cycle) = 0.
```

Comment:
nit: can you use a regular apostrophe instead of a backtick?

`s0'` instead of s0`

Reply:
done